src/tree/split_evaluator.cc

```diff
@@ -138,6 +138,15 @@ class ElasticNet final : public SplitEvaluator {
       return 0.0;
     }
   }
+  inline float ThresholdL1(float g) const {
```

Comment:
Where is this used?

Reply:
seems nowhere. Thanks, it was related to old changes when float was used in `GradStats`